test/issue-33.js

```diff
@@ -12,45 +12,32 @@ const STATE = path.join(BASE, '.migrate')
 var A1 = ['1-up', '2-up', '3-up']
 var A2 = A1.concat(['3-down', '2-down', '1-down'])
 
-describe('issue #33', function () {
+describe('down command twice', function () {
   var set
 
-  beforeEach(function (done) {
-    migrate.load({
+  beforeEach(async function () {
+    set = await migrate.load({
       stateStore: STATE,
       migrationsDirectory: BASE
-    }, function (err, s) {
-      set = s
-      done(err)
     })
   })
 
-  it('should run migrations in the correct order', function (done) {
-    set.up(function (err) {
-      assert.ifError(err)
-      assert.deepEqual(db.issue33, A1)
+  it('should run migrations in the correct order', async function () {
+    await set.up()
+    assert.deepEqual(db.issue33, A1)
 
-      set.up(function (err) {
-        assert.ifError(err)
-        assert.deepEqual(db.issue33, A1)
+    await set.up()
+    assert.deepEqual(db.issue33, A1)
 
-        set.down(function (err) {
-          assert.ifError(err)
-          assert.deepEqual(db.issue33, A2)
+    await set.down()
+    assert.deepEqual(db.issue33, A2)
 
-          set.down(function (err) {
-            assert.ifError(err)
-            assert.deepEqual(db.issue33, A2)
-
-            done()
-          })
-        })
-      })
-    })
+    await set.down()
+    assert.deepEqual(db.issue33, A2)
   })
 
-  afterEach(function (done) {
+  afterEach(function () {
     db.nuke()
-    fs.unlink(STATE, done)
+    fs.unlinkSync(STATE)
```

Comment:
```
const fs = require('fs')
const util = require('util')

// ...

const unlink = util.promisify(fs.unlink)

// ...

await unlink(STATE)
```

Reply:
i could have used bluebird too, but this is in a test and was not of a major point for me to solve that. Support my point about loads of improvements are out there.

Reply:
`util.promisify` is built into Node 🙌 no need to install any libraries

Reply:
True. Actually added bluebird to use it for another purpose which was solved differently, i like the ease of `promisifyAll` but it adds a bunch of dependencies.